### PR TITLE
Add source to positive tested choropleth on topical page

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard14**
Initially, I wanted to use the `metadata` component but I think it is a bit too tied to the tiles.

Screenshot 2021-03-18 at 09 56 12

### CODE REVIEW

**Review by VWSCoronaDashboard1 — APPROVED** [review]

File: packages/app/src/pages/actueel/gemeente/[code].tsx [hunk 1]
```diff
@@ -387,6 +388,26 @@ const TopicalMunicipality: FCWithLayout<typeof getStaticProps> = (props) => {
...
+                        siteText.positief_geteste_personen.bronnen.rivm
+                          .aria_text
+                      }
+                      href={
+                        siteText.positief_geteste_personen.bronnen.rivm.href
+                      }
+                    >
+                      {siteText.positief_geteste_personen.bronnen.rivm.text}
+                    </ExternalLink>
+                  </Text>
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/pages/actueel/gemeente/[code].tsx:410]: It looks like this part ideally should be captured in a reusable component. It's a lot of details to place directly on the root of a page layout.

File: packages/app/src/pages/actueel/veiligheidsregio/[code].tsx [hunk 2]
```diff
@@ -366,6 +367,26 @@ const TopicalSafetyRegion: FCWithLayout<typeof getStaticProps> = (props) => {
...
+                        siteText.positief_geteste_personen.bronnen.rivm
+                          .aria_text
+                      }
+                      href={
+                        siteText.positief_geteste_personen.bronnen.rivm.href
+                      }
+                    >
+                      {siteText.positief_geteste_personen.bronnen.rivm.text}
+                    </ExternalLink>
+                  </Text>
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/pages/actueel/veiligheidsregio/[code].tsx:389]: And if you're repeating the same thing again, then even more so.

File: packages/app/src/pages/index.tsx [hunk 3]
```diff
@@ -342,6 +343,27 @@ const Home: FCWithLayout<typeof getStaticProps> = (props) => {
...
+                          .aria_text
+                      }
+                      href={
+                        siteText.positief_geteste_personen.bronnen.rivm.href
+                      }
+                    >
+                      {siteText.positief_geteste_personen.bronnen.rivm.text}
+                    </ExternalLink>
+                  </Text>
+
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/pages/index.tsx:366]: So yes, if you're repeating yourself 3 times I think that's the threshold. Then it should definitely be refactored into a component.

**Review by VWSCoronaDashboard8 — CHANGES_REQUESTED** [review]

**Comment by VWSCoronaDashboard14** [comment]
@VWSCoronaDashboard8 I've integrated this in the `metadata` component now

File: packages/app/src/components-styled/metadata.tsx [hunk 4]
```diff
@@ -28,16 +36,31 @@ export function Metadata({ date, source, obtained }: MetadataProps) {
...
+              ` ${replaceVariablesInText(siteText.common.metadata.obtained, {
+                date: formatDateFromSeconds(obtained, 'weekday-medium'),
+              })}`}
+            {dateString && source ? ' · ' : null}
+            {source
+              ? `${siteText.common.metadata.source}: ${source.text}`
+              : null}
+          </Text>
+        </Box>
+      )}
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/components-styled/metadata.tsx:63]: There doesn't seem to be a clear reason why these two variations of metadata are combined into a single component. I don't mind this for now, but I feel that ideally it should be two separate components. Metadata (or ContentHeaderMetadata) and TileMetadata for example.

**Review by VWSCoronaDashboard8 — APPROVED** [review]
Much better 👍

**Pull request merged**